setup.py

```diff
@@ -10,32 +9,33 @@
     long_description = in_file.read()
 
 setup(
-    name="ismatec-control",
-    version="0.4.0",
-    description=(
-        "Library for driving the Ismatec Reglo ICC peristaltic pump."
-        "Communication is done over direct RS232 or through a serial server."
-    ),
-    author="Alex Ruddick",
-    author_email="a.ruddick@numat-tech.com",
-    license="GPLv3",
-    url="https://githut.com/numat/ismatec",
-    packages=["ismatec"],
+    name='ismatec',
```

Comment:
I need to use `ismatec-control` or something else, because Hein already has `ismatec` on PyPI.

Reply:
Hm, no good answer here. Maybe numat-ismatec?

Reply:
`/numat/numat-ismatec` is just silly. I went with `ismatec-control` for PyPI and kept `ismatec` as the module name.  Tried this with `ika-control` on TestPyPI and it [worked fine](https://test.pypi.org/project/ika-control/0.1.0/#description).